PetAdoption-iOS/PetAdoption-iOS/Views/PetCell.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  PetCell.swift
+//  PetAdoption-iOS
+//
+//  Created by Keli'i Martin on 8/19/16.
+//  Copyright © 2016 Code For Orlando. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+
+public class PetCell: UICollectionViewCell, ReusableView
+{
+    ////////////////////////////////////////////////////////////
+    // MARK: - IBOutlets
+    ////////////////////////////////////////////////////////////
+
+    @IBOutlet weak private var imageView: UIImageView!
+    @IBOutlet weak private var petNameLabel: UILabel!
+    @IBOutlet weak private var activityIndicator: UIActivityIndicatorView!
+
+    ////////////////////////////////////////////////////////////
+    // MARK: - Properties
+    ////////////////////////////////////////////////////////////
+
+    private var request: Request?
+    private var pet: Pet?
+
+    ////////////////////////////////////////////////////////////
+    // MARK: - UICollectionViewReusableView
+    ////////////////////////////////////////////////////////////
+
+    override public func prepareForReuse()
+    {
+        super.prepareForReuse()
+        imageView.image = UIImage(named: "placeholder")
```

Comment:
make sure to use self. when referring to a property in the class. Same goes for the code below.